src/muiComponents/layout/Container/index.tsx

```diff
@@ -1,38 +1,14 @@
-import { AppBar, Toolbar } from "@mui/material"
 import React, { PropsWithChildren, ReactElement } from "react"
-import {
-  ThemeMode,
-  useThemeSettings,
-} from "../../../providers/ThemeSettingsProvider"
 import MuiContainer from "@mui/material/Container/Container"
-import { Switch } from "@mui/material"
+import TopMenu from "../../../components/TopMenu"
 
 export default function Container({
   children,
 }: PropsWithChildren<unknown>): ReactElement {
-  const { themeMode, onChangeMode } = useThemeSettings()
-  const currentThemeSelected = themeMode === "dark" ?? "light"
-  const handleSwitchChange = (
-    event: React.ChangeEvent<HTMLInputElement>,
-    checked: boolean,
-  ) => {
-    const mode: ThemeMode = checked ? "dark" : "light"
-    onChangeMode(mode)
-  }
-
   return (
-    <MuiContainer maxWidth="lg">
-      <AppBar elevation={2}>
-        <Toolbar>
-          Saddle
-          <Switch
-            defaultChecked={currentThemeSelected}
-            onChange={handleSwitchChange}
-          />
-        </Toolbar>
-      </AppBar>
-      <Toolbar sx={{ marginBottom: "10px" }} />
-      {children}
-    </MuiContainer>
+    <div>
+      <TopMenu />
+      <MuiContainer maxWidth="lg">{children}</MuiContainer>
```

Comment:
This file name is a bit too generic with what it's describing. I think just wrapping the `<Switch>` in App.jsx and putting toolbar in a fragment

```
<>
   <TopMenu>
   <MuiContainer> .... </MuiContainer>
</>
```
Would be simpler and explain just as clearly.

Reply:
removed container and moved TopMenu on the App.tsx.